src/luascript.cpp

```diff
@@ -10374,56 +10386,56 @@ int LuaScriptInterface::luaPlayerSetPremiumEndsAt(lua_State* L)
 	return 1;
 }
 
-int LuaScriptInterface::luaPlayerHasBlessing(lua_State* L)
+int LuaScriptInterface::luaPlayerAddBlessing(lua_State* L)
```

Comment:
This can be written in pure Lua now, manipulating the storages.

Reply:
okay... should I update the `int LuaScriptInterface::luaPlayerSetStorageValue(lua_State* L)` method to ignore blessings range?

```cpp
if (IS_IN_KEYRANGE(key, RESERVED_RANGE)) {
	reportErrorFunc(L, fmt::format("Accessing reserved range: {:d}", key));
	pushBoolean(L, false);
	return 1;
}
```

Reply:
No, if you do that it will be impossible to set them anyway :P

Reply:
Ah sorry, I thought you were asking if the blessings storage range should be added to the protected list, not the other way around. Yes, it should be removed from the reserved range, since there is a key to access them that is very clear (`PlayerStorageKeys.blessingsRangeStart`).